tests/unit/single_table/efficacy/detection/test_detection.py

```diff
@@ -0,0 +1,64 @@
+from unittest.mock import patch
```

Comment:
Since this is testing the `sdmetrics/single_table/detection/base.py` file, let's move this to a file at the location `tests/unit/single_table/detection/test_base.py`.

Reply:
Done, just I have an error when the filename is `test_base.py` so I let `test_detection.py`. Therefore it is `test/unit/signle_table/detection/test_detection.py` .